app/stacks/global/front-door/waf-policy.tf

```diff
@@ -362,3 +363,188 @@ resource "azurerm_frontdoor_firewall_policy" "appeals_frontend" {
 
   tags = var.common_tags
 }
+
+resource "azurerm_frontdoor_firewall_policy" "back_office_applications_frontend" {
+  name                              = replace("pinswaf${local.service_name}${local.resource_suffix}", "-", "")

```

Comment:
This needs updating to be back-office applications specific.

Reply:
addressed!